### ISSUE TITLE
Make the site accessible

### ISSUE THREAD

**Issue by TheWoodenMan** [TheWoodenMan/twm-cpd-log#28]
The most common screen reader - NVDA should be able to read all text on the site.

Currently it only partially works but every part should be possible to read out with extra care taken to highlight reference numbers/text in alt tags for each entry
